crates/primitives/src/identity.rs

```diff
@@ -150,6 +150,12 @@ pub enum WalletType {
         #[serde(rename = "walletName")]
         wallet_name: String,
     },
+    INTERNETCOMPUTER {
```

Comment:
INTERNET_COMPUTER

Reply:
the thing here is that then rust throws a warning Variant `INTERNET_COMPUTER` should have CamelCase name, e.g. `InternetComputer. 
If you'd like, I can change it but then I would recommend doing it for every other variant.

Reply:
Yes, that is standard which we didn't follow 😅 you can ignore warning for now and update them separately